src/assets/styles/layout.css

```diff
@@ -1,15 +1,266 @@
+  border-top-left-radius: 0.5rem;
+  border-top-right-radius: 0.5rem;
+  height: 5%;
+  width: 100%;
+  color: rgb(0, 0, 0);
+  background-color: rgb(215, 222, 246);
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.portfolio-navigation ul {
+  display: flex;
+  flex-direction: row;
+  list-style-type: none;
+  position: relative;
+  top: 0.5rem;
+}
+
+.portfolio-navigation li {
+  margin-right: 1rem;
+}
+
+.close-btn {
+  color: rgb(237, 93, 87);
+}
+
+.minimize-btn {
+  color: rgb(249, 192, 61);
+}
+
+.maxmize-btn {
+  color: rgb(100, 206, 66);
+}
+
+main {
+  display: grid;
+  grid-template: 1fr / 1fr 4fr;
+  height: 95%;
+}
+
+.aside,
+section {
+  height: 100%;
+  width: 100%;
+}
+
+.aside {
+  background-color: rgba(59, 59, 59, 0.1);
+  border-bottom-left-radius: 0.5rem;
+}
+
+section {
+  background-color: rgb(51, 51, 51);
+  border-bottom-right-radius: 0.5rem;
+}
+
+.active {
+  background-color: rgba(72, 72, 72);
+  color: rgb(255, 255, 255);
+}
+
+.aside div {
+  margin-top: 0.2rem;
+  padding: 1rem;
+  cursor: pointer;
+  border-top-left-radius: 1rem;
+  border-bottom-left-radius: 1rem;
+  margin-left: 1rem;
+}
+
+.aside div:hover {
+  background-color: rgba(72, 72, 72, 0.5);
+  color: rgb(255, 255, 255);
+}
+
+.text-white {
+  color: var(--color-white);
+}
+
+.text-black {
+  color: var(--color-black);
+}
+
+.padding-1 {
+  padding: 1rem;
+}
+
+.scroll-y {
+  overflow-y: scroll;
+}
+
+.about-me-content {
+  width: 99%;
+  background-color: rgb(72, 72, 72);
+  margin-top: 0.2rem;
+  height: 98%;
+  display: grid;
+  grid-template: 1fr / 3fr 1fr;
+  padding: 1rem;
+  border-bottom-left-radius: 0.5rem;
+  border-bottom-right-radius: 0.5rem;
+}
+
+.about-me-content p {
+  text-align: justify;
+  line-height: 25px;
+  margin-bottom: 1rem;
+}
+
+.about-me-content img {
+  width: 15rem;
+  height: 15rem;
+  margin: 50% auto;
+  border-radius: 300rem;
+}
+
+.about-me-content-text {
+  height: 80vh;
+}
+
+.technical-skills-article {
+  width: 99%;
+  background-color: rgb(72, 72, 72);
+  margin-top: 0.2rem;
+  height: 98%;
+  padding: 1rem;
+  border-bottom-left-radius: 0.5rem;
+  border-bottom-right-radius: 0.5rem;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.technical-skills-conatiner {
+  display: grid;
+  grid-template: 1rem 4fr / 1fr;
+  row-gap: 1rem;
+  height: 50%;
+  width: 100%;
+}
+
+.technical-skills-listing {
+  display: grid;
+  grid-template: 1fr / repeat(4, 1fr);
+  column-gap: 0.5rem;
+}
+
+.technical-skills-listing img {
+  height: 100px;
+  width: 100px;
+  border-radius: 300rem;
+  margin-top: -5rem;
+}
+
+.technical-skills-listing section {
+  border: 3px solid grey;
+  border-radius: 0.5rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+  color: aliceblue;
+}
+
+.tech-skill div:not(:last-child) {
+  margin-bottom: 1rem;
+}
+
+.skill-node {
+  background-color: burlywood;
+}
+
+.skill-node div {
```

Comment:
Codacy found an issue: [Expected selector ".skill-node div" to come before selector ".aside div:hover" (no-descending-specificity)](https://app.codacy.com/manual/descholar-ceo/portfolio/pullRequest?prid=5731031)